### PR TITLE
Addressing IfcTaskTypeEnum changes on bsi repo

### PR DESCRIPTION
**Pull request by SergejMuhic**
Synchronization for IFC Tunnel and any other future MVD projects.

### CODE REVIEW

File: IFC4x3/Constants/m/MAINTENANCE_33l7ZqN8H7fvNOrgUgFUin.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="MAINTENANCE" UniqueId="c3bc78f4-5c84-47a7-95d8-d6a7aa3deb31">
+	<Documentation>Tasks required to keep an object in good working order.</Documentation>
+</DocConstant>
+
```
Comment (larswik) [inline IFC4x3/Constants/m/MAINTENANCE_33l7ZqN8H7fvNOrgUgFUin.xml]: Merge with the above MAINTENANCE constant?

**Review by pjanck — APPROVED** [review]

File: IFC4x3/Constants/m/MAINTENANCE_0_cwhoroLCy9HjJ6b6y82v.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MAINTENANCE_0_cwhoroLCy9HjJ6b6y82v" Name="MAINTENANCE" UniqueId="3e9baaf2-d725-4cf0-946d-4c6946f080b9">
-	<Documentation>Keeping something in good working order.</Documentation>
+	<Documentation>Tasks required to keep an object in good working order.</Documentation>
```
Comment (anborr) [inline IFC4x3/Constants/m/MAINTENANCE_0_cwhoroLCy9HjJ6b6y82v.xml:3]: <Documentation>Task required to keep an object in good working order.</Documentation>

Comment (SergejMuhic) [reply IFC4x3/Constants/m/MAINTENANCE_0_cwhoroLCy9HjJ6b6y82v.xml:3]: Don't shoot the messenger. This is how it is in the official repo.

**Comment by SergejMuhic** [comment]
I am merging with one outstanding comment which should probably be addressed in the bsi repo. @aothms care to have a look?
https://github.com/bSI-InfraRoom/IFC-Specification/pull/374#discussion_r896919798

**Pull request merged**